src/sphinx_book_theme/assets/styles/content/_admonitions.scss

```diff
@@ -1,20 +1,133 @@
-.admonition {
+.admonition, div.admonition {
   // Soften admonition styling from the PyData Theme
   border-radius: $box-border-radius;
   box-shadow: 0 0.2rem 0.5rem rgba(0, 0, 0, 0.05),
     0 0 0.0625rem rgba(0, 0, 0, 0.1);
-
+  padding: 0 .6rem .8rem;
+  border-color: rgba(var(--pst-color-admonition-default),1);
```

Comment:
Why do we need to be setting the border colors etc here? Couldn't we just inherit from pydata?